polynote-kernel/src/main/scala/polynote/kernel/lang/scal/ScalaSource.scala

```diff
@@ -147,6 +147,9 @@ class ScalaSource[G <: Global](
           case tree: global.MemberDef =>
             // move class/type definition to the companion object and import it within the cell's class body
             (moduleTrees :+ tree, q"import $moduleName.${tree.name}" +: classTrees)
+          case tree: global.Import =>
```

Comment:
Ah, interesting! Nice catch

Reply:
Thanks! This section isn't nearly as intimidating as it appeared at first glance! 😄 With my trusty debugger it's easy to step through. But since real Scala devs never use debuggers I guess I'm cheating 😢